cli/register/register.go

```diff
@@ -777,10 +779,23 @@ func AddAllRequiredSvcs(nodeCreds, org, url, arch, versionRange string, allRequi
 	allRequiredSvcs[svcKey].HighestVersion = highestSvc.Version // in case we don't encounter this service again, we already know the highest version for getting the user input from
 	allRequiredSvcs[svcKey].UserInputs = highestSvc.UserInputs
 
+	// check if the service contains privileged
```

Comment:
Please use exiting function to check https://github.com/open-horizon/anax/blob/63f6076e82b2d203a1d7bbb222644058ef95c944/compcheck/comp_check.go#L543

Reply:
Didn't know about that function, thanks. But we need to somehow mark result of `AddAllRequiredSvcs()` function that some of services contain privileged. So the best way I found - is marking each service separately (https://github.com/open-horizon/anax/pull/1994/files#diff-09df09ce2e6c43d6445e8d0c86f8c681R790) so that services can be checked later (https://github.com/open-horizon/anax/pull/1994/files#diff-09df09ce2e6c43d6445e8d0c86f8c681R862) . And with that function we can check if deployment contains privileged, but then we'll need to pass that result to calling function. 
Another option is to make `AddAllRequiredSvcs()` return boolean with result of `DeploymentRequiresPrivilege()`, but the name of function in this case doesn't represent that it's returning smth, and we'll have strange `if AddAllRequiredSvcs() ... {}` block.
Using of `ServicesRequirePrivilege` doesn't help as well, as we need to mark each service separately (with current implementation)
Do you have any suggestions how that could be implemented better? Maybe there is some way to get the deployment string in `CreateInputFile` itself (I can't find the possibilities now) ?

Reply:
Sorry, I do not quite follow you. Can you just replace line 783-794 with the following code?
````
if priv, err := compcheck.DeploymentRequiresPrivilege(highestSvc.Deployment, msgPrint); err != nil {
     ....
} elsie if priv {
    allRequiredSvcs[svcKey].Privileged = true
}
````

Reply:
Sorry, didn't get your request right. Updated PR